litmus-portal/frontend/src/pages/MonitoringDashboardPage/index.tsx

```diff
@@ -98,70 +96,69 @@ const DashboardPage: React.FC = () => {
     LIST_DATASOURCE,
     {
       variables: { projectID: selectedProjectID },
-      fetchPolicy: 'cache-and-network',
+      fetchPolicy: 'no-cache',
     }
   );
```

Comment:
Is it necessary to go for `no-cache` policy here? May we use `cache-and-network` policy instead

Reply:
Actually we need to check the status of data source just before loading dashboard so it’s needed so that it always makes a network call for this, same goes for the dashboard as we are updating the chaos events just before load @whitetiger1399

Reply:
Upswing cache and network here won’t reduce any load as it’s a one time fetch and also this way we can be sure of not depending on cached data for this critical use case

Reply:
with respect to `not depending on cached data` the `cache-and-network` policy executes the query and then automatically updated the cache if required which makes it provide faster response than `no-cache`. Are we checking the status of the data source every time we load the dashboard query because if we are then we need to have `cache-and-network`

Reply:
I don't think we need a cache update everytime on fetch here because we don't have that use case.